packages/clerk-js/rspack.config.js

```diff
@@ -525,31 +584,31 @@ const devConfig = ({ mode, env }) => {
     // prettier-ignore
     [variants.clerk]: merge(
       entryForVariant(variants.clerk),
-      common({ mode }),
+      common({ mode, variant: variants.clerk }),
       commonForDev(),
     ),
     // prettier-ignore
     [variants.clerkBrowser]: merge(
       entryForVariant(variants.clerkBrowser),
       isSandbox ? { entry: { sandbox: './sandbox/app.ts' } } : {},
-      common({ mode }),
+      common({ mode, variant: variants.clerkBrowser }),
       commonForDev(),
     ),
     // prettier-ignore
     [variants.clerkBrowserNoRHC]: merge(
       entryForVariant(variants.clerkBrowserNoRHC),
-      common({ mode, disableRHC: true }),
+      common({ mode, disableRHC: true, variant: variant.clerkBrowserNoRHC }),
```

Comment:
Not sure if this is correct, but it seems like it would be undefined without this change
```suggestion
      common({ mode, disableRHC: true, variant: variants.clerkBrowserNoRHC }),
```